tests/stylus/StylintBearTest.py

```diff
@@ -0,0 +1,158 @@
+import os
+from queue import Queue
+
+from bears.stylus.StylintBear import StylintBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'stylint_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as f:
+        return f.readlines()
+
+
+@generate_skip_decorator(StylintBear)
+class StylintBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = StylintBear(Section('name'), Queue())
+
+    def test_bad_missing_colon(self):
+        filename = 'test_bad_missing_colon.styl'
+        file_contents = load_testfile(filename)
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('StylintBear',
+                                message='missing colon between property '
+                                        'and value  colons',
```

Comment:
hm the message looks bad with this `colons` at the end. What does it mean anyway?